<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatCard</class>
 <widget class="QWidget" name="chatCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>64</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>64</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>64</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0" alignment="Qt::AlignVCenter">
    <widget class="QWidget" name="chatCardContainer" native="true">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>16</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="chatCardRootLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="chatIconContainer" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <widget class="QPushButton" name="onlineIndicator">
           <property name="geometry">
            <rect>
             <x>14</x>
             <y>14</y>
             <width>11</width>
             <height>11</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>11</width>
             <height>11</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>11</width>
             <height>11</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Inter</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </widget>
        </item>
        <item alignment="Qt::AlignVCenter">
         <widget class="QWidget" name="verticalLayout" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_">
           <property name="spacing">
            <number>4</number>
           </property>
           <property name="leftMargin">
            <number>8</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="firstLine">
             <property name="spacing">
              <number>2</number>
             </property>
             <item>
              <widget class="QLabel" name="chatTitle">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>16</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Montserrat</family>
                 <pointsize>10</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Заголовок</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>50</width>
                 <height>1</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="latestMessageDate">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>12</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>12</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Inter</family>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <property name="text">
                <string>12:41</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="secondLine">
             <property name="spacing">
              <number>4</number>
             </property>
             <item>
              <widget class="QLabel" name="latestMessage">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>16</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Inter</family>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <property name="text">
                <string>ОписаниеОписаниеОписаниеОписаниеОписаниеОписаниеОписание</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="unreadAmount">
               <property name="minimumSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Inter</family>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <property name="text">
                <string>3</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
